docs/io/unified_table.rst

```diff
@@ -0,0 +1,293 @@
+methods, :meth:`~astropy.table.Table.read` and :meth:`~astropy.table.Table.write`, that
+make it possible to read from and write to files. A number of formats are supported (see
+`Built-in table readers/writers`_) and new file formats and extensions can be registered
+with the :class:`~astropy.table.Table` class (see :ref:`I/O Registry <io_registry>`).
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Table Formats
+
+    unified_table_text
+    unified_table_fits
+    unified_table_hdf5
+    unified_table_parquet
+    unified_table_votable
+
+..
+  EXAMPLE START
+  Reading a DAOPhot Table
+
+To use this interface, first import the :class:`~astropy.table.Table` class,
+then call the :class:`~astropy.table.Table`
+:meth:`~astropy.table.Table.read` method with the name of the file and
+the file format, for instance ``'ascii.daophot'``:
+
+.. doctest-skip::
+
+    >>> from astropy.table import Table
+    >>> t = Table.read('photometry.dat', format='ascii.daophot')
+
+..
+  EXAMPLE END
+
+..
+  EXAMPLE START
+  Reading a Table directly from the Internet
+
+It is possible to load tables directly from the Internet using URLs. For
+example, download tables from Vizier catalogues in CDS format
+(``'ascii.cds'``):
+
+.. doctest-remote-data::
+
+    >>> from astropy.table import Table
+    >>> t = Table.read("ftp://cdsarc.u-strasbg.fr/pub/cats/VII/253/snrs.dat",
+    ...         readme="ftp://cdsarc.u-strasbg.fr/pub/cats/VII/253/ReadMe",
+    ...         format="ascii.cds")
+
+For certain file formats the format can be automatically detected, for
+example, from the filename extension::
+
+    >>> t = Table.read('table.tex')  # doctest: +SKIP
+
+..
+  EXAMPLE END
+
+..
+  EXAMPLE START
+  Writing a LaTeX Table
+
+For writing a table, the format can be explicitly specified::
+
+    >>> t.write(filename, format='latex')  # doctest: +SKIP
+
+As for the :meth:`~astropy.table.Table.read` method, the format may
+be automatically identified in some cases.
+
+The underlying file handler will also automatically detect various
+compressed data formats and uncompress them as far as
+supported by the Python installation (see
+:meth:`~astropy.utils.data.get_readable_fileobj`).
+
+For writing, you can also specify details about the `Table serialization
+methods`_ via the ``serialize_method`` keyword argument. This allows
+fine control of the way to write out certain columns, for instance
+writing an ISO format Time column as a pair of JD1/JD2 floating
+point values (for full resolution) or as a formatted ISO date string.
+
+Both the :meth:`~astropy.table.Table.read` and
+:meth:`~astropy.table.Table.write` methods can accept file paths of the form
+``~/data/file.csv`` or ``~username/data/file.csv``. These tilde-prefixed paths
+are expanded in the same way as is done by many command-line utilities, to
+represent the home directory of the current or specified user, respectively.
+
+..
+
+  EXAMPLE END
```

Comment:
Need to remove empty line or sphinx. will render the text "EXAMPLE END" in a box.

```suggestion
..
  EXAMPLE END
```